### ISSUE TITLE
Model runs with the 5, "15"-gene networks

### ISSUE THREAD

**Issue by kdahlquist** [kdahlquist/GRNmap#265]
I'm creating this issue to document the model runs that @bklein7 and @Nwilli31 will do on the 5 "15"-gene networks from #245.

For each model run, please complete a GRNmap Testing Report on OWW.  See the template here:

http://www.openwetware.org/wiki/GRNmap_Testing_Report

for what it should contain.

For starters, please run all 5 of the input workbooks to just get our feet wet running these.  Download the version 1.4.4 code from the GRNmap download page and run it from source in Matlab 2014b.

**Comment by bklein7** [comment]
I have completed the GRNmap model runs for dGLN3 & dHAP4. The run for dZAP1 remains in progress on the computer "paradoxus". I will retrieve the results from this run by tomorrow evening. Further, I created GRNsight models of all five networks using consistent ordering and spacing of genes, so @Nwilli31 does not need to worry about doing so for dCIN5 or wt. I will collect these results in a PowerPoint presentation and upload them later in the week.

**Comment by kdahlquist** [comment]
The dZAP1 run finished; I've left the computer "as is" so @bklein7 can retrieve the results later.  Looking forward to seeing the PowerPoints!

**Comment by bklein7** [comment]
I have retrieved the dZAP1 run results. @Nwilli31 and I created weighted networks in GRNsight using the same protocol as was used for the unweighted networks. We compiled the preliminary results from our runs in this [PowerPoint presentation](https://github.com/kdahlquist/DahlquistLab/blob/master/documents/NEW_BK_GRNmap_Testing_network_presentation-all-strains_20161030.pptx).

**Comment by bklein7** [comment]
@Nwilli31 and I updated the [PowerPoint presentation](https://github.com/kdahlquist/DahlquistLab/blob/master/documents/NEW_BK_GRNmap_Testing_network_presentation-all-strains_20161030.pptx) to reflect the changes requested by @kdahlquist. Specifically, the following adjustments were made:

- The slide backgrounds are now white.
- Two slides were added in which all five unweighted and weighted networks are visible together.
- The number of genes and edges for each network is displayed when introducing the network.
- The LSE : min LSE ratio is now displayed with the other parameter outputs for each network.
- The degree distribution chart generator was used for the wt & dCIN5 networks.
- All tables report values to 4 decimal places.
